CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 - Relax syntax mapping rule restrictions to allow brace expansion #2865 (@cyqsimon)
 - Apply clippy fixes #2864 (@cyqsimon)
 - Faster startup by offloading glob matcher building to a worker thread #2868 (@cyqsimon)
+- Display which theme is the default one in basic output (no colors), see #2937 (@sblondon)
```

Comment:
We could combine these two lines into one and reference both PRs on the same line, but let's do that right before release when we need to clean up the changelog anyway.

Reply:
I'm not sure who is 'we'. Do you mean the `bat` maintainers? I can do a PR for that but I doubt it worth the time.

Reply:
Yep I mean the bat maintainers, but feel free to do a PR if you like

Reply:
I done it but the CI requires to add a line in Changelog for the PR so it's not useful. I closed the PR and you or another bat maintainers will be more efficient on this topic.